src/components/app/details/appDetails/AppSecurity.tsx

```diff
@@ -4,10 +4,11 @@ import { UseGetAppSecurityDetailsProps, UseGetAppSecurityDetailsReturnType } fro
 export const useGetAppSecurityDetails = ({
     appId,
     envId,
+    artifactId,
     installedAppId,
 }: UseGetAppSecurityDetailsProps): UseGetAppSecurityDetailsReturnType => {
     const [scanResultLoading, scanResultResponse, scanResultError, reloadScanResult] = useAsync(
-        () => getSecurityScan({ appId, envId, installedAppId }),
+        () => getSecurityScan({ appId, envId, artifactId, installedAppId }),
```

Comment:
We are only sending appId and artifactId from CIDetails so shouldRun param of useAsync will fail here